### ISSUE TITLE
filters are not working after latest deployement

### ISSUE THREAD

**Issue by ajaykush2492** [someshkar/covid19india-cluster#155]
Please find the attached screenshot
<img width="1244" alt="Screenshot 2020-03-29 at 2 34 27 PM" src="https://user-images.githubusercontent.com/54022006/77845228-c921f600-71ca-11ea-918a-b27870456ae9.png">

**Comment by sampathBlam** [comment]
We are working on it. It is because of certain patients whose state (as in statename) is unknown. 
PatientIds - 251047,251048,251049

**Comment by sampathBlam** [comment]
It is because some of the patients do not have value for state ( name of the state) 
PatientIds : 
251047
251048
251049

**Comment by ajaykush2492** [comment]
Hi Team,
can i get pull request approved ao i can try to fix it?
Thanks & Regards
Ajay

**Comment by sampathBlam** [comment]
Working on it.

**Comment by sampathBlam** [comment]
Travel filter is blocked by another issue #152

**Comment by sibeshkar** [comment]
have removed travel filter temporarily.
